extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -97,7 +97,7 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     }
 
     $scope.getMinText = function(item) {
-        if (!item.stateDescription || !item.stateDescription.minimum) {
+        if (!item.stateDescription || isNaN(item.stateDescription.minimum)) {
```

Comment:
Yes, it will work in that case. I was just not sure about empty or null being passed to sprintf when pattern is defined. It does not crash in that case as well. So all is fine.